desktop/cmp/error/ErrorMessage.ts

```diff
@@ -91,11 +118,20 @@ const messageCmp = hoistCmp.factory(({message}) => {
     return null;
 });
 
-const actionButton = hoistCmp.factory(({actionButtonProps}) => {
-    if (isEmpty(actionButtonProps)) return null;
+const actionButton = hoistCmp.factory<ButtonProps>(props => {
     return button({
         text: 'Retry',
-        minimal: false,
-        ...actionButtonProps
+        icon: Icon.refresh(),
+        minimal: true,
```

Comment:
Nit- since `minimal = true` by default, wondering why explicitly specify it here

Reply:
good nit, fixed